lib/rgeo/coord_sys/cs/entities.rb

```diff
@@ -183,16 +182,14 @@ def to_s
         def to_wkt(standard_brackets = false)
           open, close = brackets(standard_brackets)
           content = wkt_content(standard_brackets).map { |obj| ",#{obj}" }.join
-          if defined?(@authority) && @authority
-            authority = ",AUTHORITY#{open}#{@authority.inspect},#{@authority_code.inspect}#{close}"
-          else
-            authority = ""
-          end
-          if defined?(@extensions) && @extensions
-            extensions = @extensions.map { |k, v| ",EXTENSION#{open}#{k.inspect},#{v.inspect}#{close}" }.join
-          else
-            extensions = ""
-          end
+          authority =
+            defined?(@authority) &&
+            @authority &&
+            ",AUTHORITY#{open}#{@authority.inspect},#{@authority_code.inspect}#{close}"
+          extensions =
+            defined?(@extensions) &&
+            @extensions &&
+            @extensions.map { |k, v| ",EXTENSION#{open}#{k.inspect},#{v.inspect}#{close}" }.join
```

Comment:
blocking: this changes the behaviour, now `extensions` and `authority` can be false

Reply:
I'm a little confused on this one. They can either be `nil` or be a string (`,AUTHORITY` / `,EXTENSION`). It is then passed to `""`, which then would convert the `nil` to an empty string.

Since, like I said, I'm not 100% sure on all of the internals of this, so I can update this to read

```ruby
         authority =
            if defined?(@authority) && @authority
              ",AUTHORITY#{open}#{@authority.inspect},#{@authority_code.inspect}#{close}"
            else
              ""
            end
          extensions =
            if defined?(@extensions) && @extensions
              @extensions.map { |k, v| ",EXTENSION#{open}#{k.inspect},#{v.inspect}#{close}" }.join
            else
              ""
            end
```

which would be what it was before and make rubocop happy 👮‍♂️ 

Would something like that be good?

Reply:
I think it'd be better.

The current solution depends a lot on what `@extensions` is, and can give a bug similar to the one you had in previous iterations. 

The `defined?` method actually can return a string, and `@extension` could be `nil`, but `false` would then set the variable to `false`